src/navigations/root-stack.js

```diff
@@ -197,7 +197,7 @@ const AuthenticatedNavigator = () => {
         />
         <AuthenticatedStack.Screen
           name="Followings"
-          component={withSafeAreaView(withKeyboardWrapper(FollowingScreen))}
+          component={FollowingScreen}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change removes the `withSafeAreaView` and `withKeyboardWrapper` higher-order components from the `FollowingScreen`. This could potentially lead to issues with safe area view handling and keyboard interactions on the Followings screen. If these functionalities are now handled within the `FollowingScreen` component itself, then this change is fine. Otherwise, you might want to reconsider this modification.
```diff
- component={FollowingScreen}
+ component={withSafeAreaView(withKeyboardWrapper(FollowingScreen))}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->